tests/test_cmd_lossless.sh

```diff
@@ -40,65 +40,29 @@ fi
 AVIFDEC="${BINARY_DIR}/avifdec"
 ARE_IMAGES_EQUAL="${BINARY_DIR}/tests/are_images_equal"
-
-# Basic calls.
-"${AVIFENC}" --version
-"${AVIFDEC}" --version
+IGNORE_ALPHA_FLAG=1
 
 # Input file paths.
-INPUT_Y4M="${TESTDATA_DIR}/kodim03_yuv420_8bpc.y4m"
 INPUT_PNG="${TESTDATA_DIR}/paris_icc_exif_xmp.png"
-INPUT_JPG="${TESTDATA_DIR}/paris_exif_xmp_icc.jpg"
 # Output file names.
 ENCODED_FILE="avif_test_cmd_encoded.avif"
-ENCODED_FILE_NO_METADATA="avif_test_cmd_encoded_no_metadata.avif"
-ENCODED_FILE_WITH_DASH="-avif_test_cmd_encoded.avif"
 DECODED_FILE="avif_test_cmd_decoded.png"
-DECODED_FILE_LOSSLESS="avif_test_cmd_decoded_lossless.png"
 
 # Cleanup
 cleanup() {
   pushd ${TMP_DIR}
-    rm -- "${ENCODED_FILE}" "${ENCODED_FILE_NO_METADATA}" \
-          "${ENCODED_FILE_WITH_DASH}" "${DECODED_FILE}" "${DECODED_FILE_LOSSLESS}"
+    rm -- "${ENCODED_FILE}" "${DECODED_FILE}"
   popd
 }
 trap cleanup EXIT
 
 pushd ${TMP_DIR}
-  # Lossy test. The decoded pixels should be different from the original image.
-  echo "Testing basic lossy"
-  "${AVIFENC}" -s 8 "${INPUT_Y4M}" -o "${ENCODED_FILE}"
-  "${AVIFDEC}" "${ENCODED_FILE}" "${DECODED_FILE}"
-  "${ARE_IMAGES_EQUAL}" "${INPUT_Y4M}" "${DECODED_FILE}" 0 && exit 1
-
   # Lossless test. The decoded pixels should be the same as the original image.
   echo "Testing basic lossless"
-  # TODO(yguyon): Make this test pass with INPUT_PNG instead of DECODED_FILE.
-  "${AVIFENC}" -s 10 -l "${DECODED_FILE}" -o "${ENCODED_FILE}"
-  "${AVIFDEC}" "${ENCODED_FILE}" "${DECODED_FILE_LOSSLESS}"
-  "${ARE_IMAGES_EQUAL}" "${DECODED_FILE}" "${DECODED_FILE_LOSSLESS}" 0
-
-  # Argument parsing test with filenames starting with a dash.
-  echo "Testing arguments"
-  "${AVIFENC}" -s 10 "${INPUT_PNG}" -- "${ENCODED_FILE_WITH_DASH}"
-  "${AVIFDEC}" --info  -- "${ENCODED_FILE_WITH_DASH}"
-  # Passing a filename starting with a dash without using -- should fail.
-  "${AVIFENC}" -s 10 "${INPUT_PNG}" "${ENCODED_FILE_WITH_DASH}" && exit 1
-  "${AVIFDEC}" --info "${ENCODED_FILE_WITH_DASH}" && exit 1
-
-  # Metadata test.
-  echo "Testing metadata enc/dec"
-  for INPUT in "${INPUT_PNG}" "${INPUT_JPG}"; do
-    "${AVIFENC}" "${INPUT}" -o "${ENCODED_FILE}"
-    # Ignoring a metadata chunk should produce a different output file.
-    "${AVIFENC}" "${INPUT}" -o "${ENCODED_FILE_NO_METADATA}" --ignore-icc
-    cmp "${ENCODED_FILE}" "${ENCODED_FILE_NO_METADATA}" && exit 1
-    "${AVIFENC}" "${INPUT}" -o "${ENCODED_FILE_NO_METADATA}" --ignore-exif
-    cmp "${ENCODED_FILE}" "${ENCODED_FILE_NO_METADATA}" && exit 1
-    "${AVIFENC}" "${INPUT}" -o "${ENCODED_FILE_NO_METADATA}" --ignore-xmp
-    cmp "${ENCODED_FILE}" "${ENCODED_FILE_NO_METADATA}" && exit 1
-  done
+  # TODO(yguyon): Make this test pass with IGNORE_ALPHA_FLAG set to 0.
+  "${AVIFENC}" -s 10 -l "${INPUT_PNG}" -o "${ENCODED_FILE}"
+  "${AVIFDEC}" "${ENCODED_FILE}" "${DECODED_FILE}"
+  "${ARE_IMAGES_EQUAL}" "${DECODED_FILE}" "${DECODED_FILE}" "${IGNORE_ALPHA_FLAG}"
```

Comment:
IMPORTANT: Both arguments are the same (`${DECODED_FILE}`). One of the arguments should be changed to `${INPUT_PNG}`.

It would be good for this pull request to focus on fixing https://github.com/AOMediaCodec/libavif/issues/1175 and not attempt to fix the original TODO `TODO(yguyon): Make this test pass with INPUT_PNG instead of DECODED_FILE`.